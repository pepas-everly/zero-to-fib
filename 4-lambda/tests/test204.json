../../3-if/tests/test204.json