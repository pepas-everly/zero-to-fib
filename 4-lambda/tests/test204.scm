../../3-if/tests/test204.scm