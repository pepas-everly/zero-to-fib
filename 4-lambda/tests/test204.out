../../3-if/tests/test204.out